include/exadg/time_integration/ab_constants.cpp

```diff
@@ -0,0 +1,158 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+// deal.II
+#include <deal.II/base/exceptions.h>
+
+// ExaDG
+#include <exadg/time_integration/ab_constants.h>
+
+namespace ExaDG
+{
+ABTimeIntegratorConstants::ABTimeIntegratorConstants(unsigned int const order,
+                                                     bool const         start_with_low_order)
+  : TimeIntegratorConstantsBase(order, start_with_low_order), alpha(order)
+{
+  // In case of  ABM it is possible to have AB of order 0
```

Comment:
Meaning of "ABM" is unclear. 

What do you mean by "order 0"?

Reply:
Adams--Bashforth--Moulton predictor-corrector methods are also called ABM. I will add a comment.

Order 0 means, that the prediction (usually obtained from AB) is the solution of the last time step. To be able to write generic code, I would like to be able to initialize the constants with `oder=0` (which means, that no coefficients are managed). It is hard to explain this with just the constants. If it is OK for you, I would keep it this way and comment on this in a follow-up PR in which the actual time integrators are introduced.